docs/src/main/paradox/other-docs.md

```diff
@@ -1,5 +1,8 @@
 # Other documentation
 
+[From a time-series database to a key operational technology for the enterprise](https://youtu.be/3APiIht6oDY?t=1298)
```

Comment:
remove the ?t= though, people want to watch from the start ;)

Reply:
Nooo, I put it there intentionally to point to the "Integration" part. It is easier to find the start than the Alpakka-explaining section.

Reply:
Oh, please mention such in PR next time :-) I'll add it back, sorry :)

Reply:
Sorry, for not pointing it out.